### PR TITLE
Setup Lint, Prettier, Commonlint, ESLint

### PR DESCRIPTION
**Pull request by dhanush027**
A basic setup for checking code format.

### CODE REVIEW

**Comment by dhanush027** [comment]
Closes https://github.com/Real-Dev-Squad/website-calendar/issues/6

File: app/tailwind.css [hunk 1]
```diff
@@ -34,15 +38,22 @@
...
-  -moz-tab-size: 4;
+  tab-size: 4;
+
   /* 3 */
-  -o-tab-size: 4;
-     tab-size: 4;
+  tab-size: 4;
+  tab-size: 4;
+
   /* 3 */
```
Comment (rohan09-raj) [inline app/tailwind.css:52]: Can't figure out why `Number` comments are all over the CSS files.
@harshith-venkatesh @dhanush027 What's the use of adding these comments ??

File: app/tailwind.css [hunk 2]
```diff
@@ -423,49 +468,51 @@ Ensure the default browser behavior of the `hidden` attribute.
...
-  --tw-numeric-spacing:  ;
-  --tw-numeric-fraction:  ;
-  --tw-ring-inset:  ;
-  --tw-ring-offset-width: 0px;
+  --tw-ordinal: ;
+  --tw-slashed-zero: ;
+  --tw-numeric-figure: ;
+  --tw-numeric-spacing: ;
+  --tw-numeric-fraction: ;
+  --tw-ring-inset: ;
```
Comment (rohan09-raj) [inline app/tailwind.css:490]: No values are provided to these properties.

File: app/tailwind.css [hunk 3]
```diff
@@ -423,49 +468,51 @@ Ensure the default browser behavior of the `hidden` attribute.
...
+  --tw-drop-shadow: ;
+  --tw-backdrop-blur: ;
+  --tw-backdrop-brightness: ;
+  --tw-backdrop-contrast: ;
+  --tw-backdrop-grayscale: ;
+  --tw-backdrop-hue-rotate: ;
+  --tw-backdrop-invert: ;
+  --tw-backdrop-opacity: ;
+  --tw-backdrop-saturate: ;
+  --tw-backdrop-sepia: ;
```
Comment (rohan09-raj) [inline app/tailwind.css:515]: Here too

File: app/tailwind.css (hunk 1 above)
Comment (dhanush027) [reply app/tailwind.css:52]: I haven't modified the file,but i think it was used to give some reference.

File: app/tailwind.css (hunk 2 above)
Comment (dhanush027) [reply app/tailwind.css:490]: I only did the lint, prettier formatting, but didn't modify any code.

File: app/tailwind.css (hunk 3 above)
Comment (dhanush027) [reply app/tailwind.css:515]: Yes, values weren't set previously.
I only did lint and prettier check,

**Pull request merged**